app/controllers/api_controller/parameters.rb

```diff
@@ -28,26 +28,38 @@ def hash_fetch(hash, element, default = {})
     def filter_param(klass)
       return nil if params['filter'].blank?
 
-      operators = {"!=" => {:default => :not_eq, :str => :does_not_match},
+      operators = {"!=" => {:default => :not_eq, :str => :does_not_match, :ruby => :!=},
                    "<=" => {:default => :lteq},
                    ">=" => {:default => :gteq},
                    "<"  => {:default => :lt},
                    ">"  => {:default => :gt},
-                   "="  => {:default => :eq, :str => :matches}
```

Comment:
Can't the ruby_filters be used for numerical comparators, i.e. virtual attributes returning integers for examples ?
If so, we could define :ruby for the remaining <=, >=, < and >.